tests/integration/test_arctic.py

```diff
@@ -190,3 +194,101 @@ def test_lib_rename_namespace(arctic):
 def test_lib_type(arctic):
     arctic.initialize_library('test')
     assert(arctic.get_library_type('test') == VERSION_STORE)
+
+
+MY_ARCTIC = None  # module-level Arctic singleton
```

Comment:
might be worth splitting into a test_arctic_multithread.py

Reply:
sure, splitted